src/simpleeventemitter.js

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
Why not having this private class inside `watchdog.js`? We do that in a lot of places. Honestly, I expected to keep it simple and just have those few methods straight in the `Watchdog` class. I am not sure if we will be using this anywhere else, especially since we want to keep it in this repo.

Reply:
I'll move these method directly to the watchdog class.

Reply:
Fixed.